test/e2e/remediation_test.go

```diff
@@ -99,10 +100,14 @@ func remediation() {
 			Logf("Error: %v", err)
 			return err
 		}
-		g.Expect(filterBmhsByProvisioningState(bmhs, bmh.StateAvailable)).To(HaveLen(newReplicaCount + len(workerM3Machines)))
+		g.Expect(filterBmhsByProvisioningState(bmhs, bmo.StateAvailable)).To(HaveLen(newReplicaCount + len(workerM3Machines)))
 		return nil
 	}, e2eConfig.GetIntervals(specName, "wait-machine-remediation")...).Should(Succeed())
 
+	// Calling an inspection tests here for now until we have a parallelism enabled in e2e framework.
+	inspection()
+
+	Logf("Start checking unhealthy annotation")
```

Comment:
A similar Log entry before you call inspection() ? Something like `Starting re-introspection test` ?

Reply:
Thanks for reviewing @kashifest . That log is already inside the inspection() call, see here https://github.com/metal3-io/cluster-api-provider-metal3/pull/483/files#diff-968216775903e52091aa51d52ca10af4ec608c73457ce10aaa29a9b05c82bb0bR17

Reply:
Ow. ok. All good then